rendermodules/montage/run_montage_job_for_section.py

```diff
@@ -215,36 +143,19 @@ def run(self):
         else:
             raise RenderModuleException("solve failed with input_json {}",self.args)
 
-        sectionDataList = renderapi.stack.get_stack_sectionData(self.args['target_collection']['stack'],
-            render=self.render)
-        sectionData = next(section for section in sectionDataList if section['z']==self.args['first_section'])
-        self.output(sectionData)
-
-        '''
-        if os.path.isfile(self.solver_executable) and os.access(self.solver_executable, os.X_OK):
-            cmd_to_qsub = "%s %s"%(self.solver_executable, tempjson.name)
-
-            #generate pbs file
-            temppbs = tempfile.NamedTemporaryFile(
-                suffix=".pbs",
-                mode="w",
-                delete=False)
-            temppbs.close()
-
-            with open(temppbs.name, 'w') as f:
-                f.write('#PBS -l mem=60g\n')
-                f.write('#PBS -l walltime=00:30:00\n')
-                f.write('#PBS -l ncpus=1\n')
-                f.write('#PBS -N Montage\n')
-                f.write('#PBS -r n\n')
-                f.write('#PBS -m n\n')
-                f.write('#PBS -q emconnectome\n')
-                f.write('%s\n'%(cmd_to_qsub))
-            f.close()
-
-            qsub_cmd = 'qsub %s'%(temppbs.name)
-            subprocess.call(qsub_cmd)
-        '''
+        #return the section data as the output_json if it was given
+        if self.args.get('output_json',None) is not None:
+            sectionDataList = renderapi.stack.get_stack_sectionData(self.args['target_collection']['stack'],
```

Comment:
Is the solver setting the target collection to complete?  That was configurable, but I don't know if it made it into the dev branch.  I think this will break on LOADING stacks prohibiting continuous processing.

Reply:
I'm not sure.. but I think that's a change we should probably implement in the Solver, as we would have problems if the solver closed the stack and then this module reset it to loading.. that might leave some spare moments of closed stack that could cause problems, so i'm not going to make any changes here.

Reply:
It was an option in the old version of the solver, but not the new one. But that can be changed easily. I will make that change internal to the solver... Which then requires this module to take that option as input as well. Or the other way is to leave the stack in LOADING state by default from the solver side.

Reply:
I think we should make the solver take this as an option, and then we make the default value in the schema here to be to leave it Loading.

Reply:
can we go ahead and approve this PR though? this change is needed for beta1 and having the stack in a complete state won't break future stacks from solving into it, as the solver sets it to loading before uploading results.

Reply:
In the event of concurrent montaging having individual montage solves setting the stack to COMPLETE should prevent others which are in the process of uploading tiles, right?

Reply:
yes you are right.. i've made the change to the modules and added the option in the solver.